streams-contrib/streams-persist-graph/src/main/java/org/apache/streams/graph/GraphHttpPersistWriter.java

```diff
@@ -218,7 +217,7 @@ protected ObjectNode executePost(HttpPost httpPost) {
     @Override
     public void prepare(Object configurationObject) {
```

Comment:
I created https://issues.apache.org/jira/browse/STREAMS-375 to address this project-wide.